Source/IGListAdapterUpdaterDelegate.h

```diff
@@ -37,7 +37,9 @@ NS_ASSUME_NONNULL_BEGIN
 
  @note This event is called in the completion block of the batch update.
  */
-- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater didPerformBatchUpdates:(IGListBatchUpdateData *)updates withCollectionView:(UICollectionView *)collectionView;
+- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater
+    didPerformBatchUpdates:(IGListBatchUpdateData *)updates
+        withCollectionView:(UICollectionView *)collectionView;
```

Comment:
nit:

this is the only delegate method spelled `withCollectionView:`. 😞  the others are simply spelled `collectionView:`.

I'd prefer to remove the "with" here. (could be another diff. will require changelog entry)